src/main/resources/application-rh-marketplace.yaml

```diff
@@ -11,3 +11,9 @@ rhsm-subscriptions:
     verify-batches: ${RH_MARKETPLACE_VERIFY_BATCHES:true}
     manual-marketplace-submission-enabled: ${RH_MARKETPLACE_MANUAL_SUBMISSION_ENABLED:false}
     amendment-not-supported-marker: ${RH_MARKETPLACE_AMENDMENT_NOT_SUPPORTED_MARKER:(amendments) is not available}
+    billable-usage:
+      incoming:
+        topic: platform.rhsm-subscriptions.billable-usage
+        kafka-group-id: swatch-producer-rh-marketplace
+        seek-override-end: ${RH_MARKETPLACE_BILLABLE_USAGE_KAFKA_SEEK_OVERRIDE_END:false}
```

Comment:
I think these new env vars need to be added to templates/swatch-billing-provider-rh-marketplace.yml and rhsm-clowdapp.yaml.